pom.xml

```diff
@@ -2,83 +2,167 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <modelVersion>4.0.0</modelVersion>
+    <modelVersion>4.0.0</modelVersion>
 
-  <groupId>com.amazonaws</groupId>
-  <artifactId>amazon-sqs-java-extended-client-lib</artifactId>
-  <version>1.0.0</version>
-  <packaging>jar</packaging>
-  <name>Amazon SQS Extended Client Library for Java</name>
-  <description>An extension to the Amazon SQS client that enables sending and receiving messages up to 2GB via Amazon S3.
-  </description>
-  <url>https://github.com/awslabs/amazon-sqs-java-extended-client-lib/</url>
+    <groupId>com.amazonaws</groupId>
+    <artifactId>amazon-sqs-java-extended-client-lib</artifactId>
+    <version>1.0.1.LAMPO</version>
+    <packaging>jar</packaging>
+    <name>Amazon SQS Extended Client Library for Java</name>
+    <description>An extension to the Amazon SQS client that enables sending and receiving messages up to 2GB via Amazon
+        S3.
+    </description>
+    <url>https://github.com/awslabs/amazon-sqs-java-extended-client-lib/</url>
 
-  <scm>
-    <url>https://github.com/awslabs/amazon-sqs-java-extended-client-lib.git</url>
-  </scm>
+    <scm>
+        <url>https://github.com/awslabs/amazon-sqs-java-extended-client-lib.git</url>
+    </scm>
```

Comment:
Should we update these URLs to point to our fork? Or better to keep it pointing to the original?